tests/api/runtimes/test_spark.py

```diff
@@ -622,8 +626,26 @@ def test_get_offline_features(
             },
             "outputs": [],
             "output_path": "v3io:///mypath",
-            "function": "None/my-vector_merger@0f4fef1da6f72c229b33fefbff0e5b58d87263c7",
+            "function": "None/my-vector_merger@b1bb6dd86fd4eb95cff8f6231b260dcc71fbeaa0",
             "secret_sources": [],
             "data_stores": [],
             "handler": "merge_handler",
         }
+
+        self.name = "my-vector_merger"
+        self.project = "default"
+        self._assert_custom_object_creation_config()
+
+        expected_code = base64.b64encode(
+            _default_merger_handler.replace(
+                "{{{engine}}}", "SparkFeatureMerger"
+            ).encode("UTF-8")
+        ).decode("UTF-8")
+
+        body = self._get_custom_object_creation_body()
+        code = None
+        for envvar in body["spec"]["driver"]["env"]:
+            if envvar["name"] == "MLRUN_EXEC_CODE":
+                code = envvar["value"]
+                break
+        assert code == expected_code
```

Comment:
I would move this part into `_assert_custom_object_creation_config` - pass some `expected_code` or something.